### PR TITLE
Feat/exchange rate graph 81

### LINKED ISSUE #81: Exchange  rate graph
Let's make an exchange rate, through time and presented on a graphic

we can try and make a line graph appear over the stacked bar graph with a separate y axis 
image

If this is too complicated we can make twi separate graphs and have a line graph just for exchange rate

### LINKED ISSUE #107: Replace "REACT_APP_NAME" on .env.example
In the file .env.example, on webapp replace "REACT_APP_NAME" with "0.0.1"

### PR DESCRIPTION
**Pull request by JAMares**
### What does this PR do?

- Resolve #81 
- Resolve #107 

### Steps to test

1. Run project 
2. Go to Income
3. You should see, the next: 
imagen

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -30,7 +30,7 @@ HAPI_RATE_HISTORY=https://api.coingecko.com/api/v3
 #webapp
 PORT=3000
 REACT_APP_TAG=localdev
-REACT_APP_NAME=$npm_package_name
+REACT_APP_NAME=0.0.1
```
Comment (xavier506) [inline .env.example:33]: can also be set to `local` instead of a version number

File: .env.example [hunk 2]
```diff
@@ -30,7 +30,7 @@ HAPI_RATE_HISTORY=https://api.coingecko.com/api/v3
...
 REACT_APP_TITLE=Eden on EOS Accounting Standard
```
Comment (xavier506) [inline .env.example:34]: set title to : `Eden on EOS delegate spend explorer`

File: .env.example [hunk 3]
```diff
@@ -30,7 +30,7 @@ HAPI_RATE_HISTORY=https://api.coingecko.com/api/v3
...
 REACT_APP_LOGO=https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosCR/fullColor-vertiall-transparent-white.png
```
Comment (xavier506) [inline .env.example:35]: are we using this en var?

Comment (JAMares) [reply .env.example:35]: i dont know, but if i use `$npm_package_name` apear several warnings and on workflows appear like `eosreports`

Comment (JAMares) [reply .env.example:35]: Ok, i deleted it and work fine, so i erase it?

**Review by tetogomez — APPROVED** [review]
LGTM

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**